### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by 4812571**

### CODE REVIEW

**Comment by 4812571** [comment]
I believe the commits could be better on this, but I don't know how to rewrite git history.

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google Inc.
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Needs a package stage and you can remove the copyright.

File: mayhem/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
```
Comment (xansec) [inline mayhem/build.sh:1]: No need for a script for just one line.

**Review by xansec — CHANGES_REQUESTED** [review]
These look a lot like OSS-Fuzz integrations? Can you explain? Btw, you can "squash" your commits to a single commit before making an MR.  Please make the requested changes.

**Comment by 4812571** [comment]
I used OSS-Fuzz conventions for my pre-harnessed Go repositories, because it was a very reusable and modular convention in my opinion. That is why there's a copyright and a build script.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**